### PR TITLE
fix docs warnings

### LINKED ISSUE #540: Some issues to fix in the compilation of docs
The docs currently compile but with some warnings (sphinx 5.0)

```
WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid langauge code. Falling back to 'en' (English).
making output directory... done
WARNING: extlinks: Sphinx-6.0 will require a caption string to contain exactly one '%s' and all other '%' need to be escaped as '%%'.
WARNING: extlinks: Sphinx-6.0 will require a caption string to contain exactly one '%s' and all other '%' need to be escaped as '%%'.
```

### PR DESCRIPTION
**Pull request by MaskDuck**
## Summary

close #540 

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: nextcord/application_command.py [hunk 1]
```diff
@@ -2038,6 +2038,7 @@ def user_command(
 ):
     """Creates a User context command from the decorated function.
     Used inside :class:`ClientCog`'s or something that subclasses it.
+
```
Comment (MaskDuck) [inline nextcord/application_command.py:2041]: maybe it fixes it but im not really sure.

Comment (MaskDuck) [reply nextcord/application_command.py:2041]: *it refers to `/home/docs/checkouts/readthedocs.org/user_builds/nextcord/checkouts/latest/nextcord/ext/commands/bot.py:docstring of nextcord.ext.commands.bot.BotBase.application_command_before_invoke:10: WARNING: Literal block ends without a blank line; unexpected unindent.` error.

File: nextcord/ext/commands/bot.py [hunk 2]
```diff
@@ -1152,9 +1152,8 @@ def application_command_before_invoke(self, coro: ApplicationHook) -> Applicatio
         connections or any type of set up required.
         This pre-invoke hook takes a sole parameter, a :class:`.Interaction`.
         .. note::
-            The :meth:`.application_command_before_invoke` and :meth:`.application_command_after_invoke`
-            hooks are only called if all checks pass without error. If any check fails, then the hooks
-            are not called.
+            The :meth:`.application_command_before_invoke` and :meth:`.application_command_after_invoke` hooks are only called if all checks pass without error. If any check fails, then the hooks are not called.
```
Comment (ooliver1) [inline nextcord/ext/commands/bot.py:1155]: May I ask why such a huge line is needed, docs allow for multi line docstrings

File: nextcord/application_command.py (hunk 1 above)
Comment (ooliver1) [reply nextcord/application_command.py:2041]: "not really sure": you can build docs locally

**Comment by ooliver1** [comment]
I'm sure there are significantly better ways to fix these warnings than make insanely long line lengths

**Pull request closed without merging**